src/components/CCPDashboard/CCPDashboardPage.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { Colours } from "../../styles/Constants";
+import { Box, Tabs, Tab, makeStyles } from "@material-ui/core";
+import { RouteComponentProps } from "react-router";
+import { useAllPatients } from "../../graphql/queries/hooks/patients";
+import { PatientOverview } from "./PatientOverview";
+
+type TParams = { eventId: string; ccpId: string };
```

Comment:
small thing, `type` is used here, while `interface` is used on line 15, is there a reason we're using type over interface?

Reply:
in this particular use case, it makes no difference so i'll change it to `interface` for consistency :)